sdk/transaction.go

```diff
@@ -36,6 +32,101 @@ func (txs *TransactionService) GetTransaction(ctx context.Context, id string) (T
 	return MapTransaction(&b, txs.client.GenerationHash())
 }
 
+// returns an array of Transaction's for passed array of transaction ids or hashes
+func (txs *TransactionService) getTransactionsByGroup(ctx context.Context, group TransactionGroups, tpOpts *TransactionsPageOptions) (*TransactionsPage, error) {
+	tspDTO := &transactionsPageDTO{}
+
+	u, err := addOptions(fmt.Sprintf("%s/%s", transactionsRoute, group.String()), tpOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := txs.client.doNewRequest(ctx, http.MethodGet, u, nil, &tspDTO)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = handleResponseStatusCode(resp, map[int]error{404: ErrResourceNotFound, 409: ErrArgumentNotValid}); err != nil {
+		return nil, err
+	}
+
+	return tspDTO.toStruct(txs.client.GenerationHash())
+}
+
+// returns an array of Transaction's for passed array of transaction ids or hashes
+func (txs *TransactionService) getTransactionsByIds(ctx context.Context, group TransactionGroups, ids []string) ([]Transaction, error) {
+	var b bytes.Buffer
+	txIds := &TransactionIdsDTO{
+		ids,
+	}
+
+	resp, err := txs.client.doNewRequest(ctx, http.MethodPost, fmt.Sprintf("%s/%s", transactionsRoute, group.String()), txIds, &b)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = handleResponseStatusCode(resp, map[int]error{404: ErrResourceNotFound, 409: ErrArgumentNotValid}); err != nil {
+		return nil, err
+	}
+
+	return MapTransactions(&b, txs.client.GenerationHash())
+}
+
+// returns Transaction for passed transaction id or hash
+func (txs *TransactionService) GetTransaction(ctx context.Context, id string) (Transaction, error) {
```

Comment:
*GetAnyTransactionById will be better

Reply:
Note: That's for SDK semantics to be consistent
From Golang philosophy perspective, your original name is more preferable